README.md

```diff
@@ -177,13 +177,13 @@ domassist.html('.my-div', 'hello world'); // add html
 domassist.html('.my-div', ''); // remove html
 ```
 
-### closest(element, selector)
+### closest(origin, selector)
 
 Find the closest parent element that matches the given selector
 
 #### Parameters:
 
-`element` - {Element} the element where you want to start looking from
+`origin` - {string:Element} - the element where you want to start looking from. It can be either a valid CSS selector or an Element.
```

Comment:
Nitpicky. Looking like JSDoc here. Can we have `string|Element` (union type). Seems correct down below for `once`